setup.md

```diff
@@ -88,7 +88,7 @@ A video format of these setup instructions is available here:
 1. In another terminal, from the `guac-compose` directory, run:
 
    ```bash
-   docker compose up --force-recreate
+   docker compose -f docker-compose.yml -f container_files/mem.yaml up --force-recreate
```

Comment:
This PR cannot merge until we do another release for GUAC!

Reply:
shall we release a patch version for this?

Reply:
v0.4.0 released.